### DISCUSSION TITLE
VRoid SDK for Unreal Engine 0.0.1で取得したvrmファイル名が文字化けする

### DISCUSSION THREAD

**Discussion by koiusa** [pixiv/vroid-sdk-developers#124] [Q&A]
もしかしたら、私の環境だけかもしれないですがvroidHubで登録しているアバター名が日本語表記の場合  SDKで取得され(ProjectDir)/Saved/VRoid/vrm 配下に配置されるvrmファイル名が文字化けするようです。既知の現象でしたら恐縮です。

**Comment by ytomi-pixiv** [comment]
モデル名が日本語の場合にファイル名が文字化けする件、手元でも確認できました。
まだ原因調査の段階ですが、今後のアップデートで対応できればと思います。報告ありがとうございました！
